proto/ibc/applications/middleware/fee/v1/genesis.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package ibc.applications.middleware.fee.v1;
+
+import "gogoproto/gogo.proto";
+import "ibc/applications/middleware/fee/v1/fee.proto";
+import "ibc/core/channel/v1/channel.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
+
+// GenesisState defines the fee middleware genesis state
+message GenesisState {
+  // An entry for the PacketsFees map
+  message PacketsFeesMapEntry {
```

Comment:
Let's make this `IdentifiedPacketFee` and defined outside of genesis. This follows our existing pattern

Reply:
Still inside genesis.proto just outside of GenesisState correct? Or do you mean outside genesis.proto?

Reply:
I would put it in `fee.proto` to be consistent what we [already do](https://github.com/cosmos/ibc-go/blob/0afa15af41997cfb4c5dd5f98de8043961e26195/proto/ibc/core/channel/v1/channel.proto#L31). This will be used by query.proto as well

Reply:
Updated